test_wazuh/test_fim/test_ambiguous_confs/data/wazuh_conf_simple.yaml

```diff
@@ -91,6 +91,44 @@
       attributes:
         - FIM_MODE
 # conf 6
+- tags:
+  - ambiguous_recursion_tag_over
+  apply_to_modules:
+  - test_ambiguous_simple
+  section: syscheck
+  elements:
+  - disabled:
+      value: 'no'
+  - directories:
+      value: "/recursiondir_tag"
+      attributes:
+        - recursion_level: "2"
+        - tags: TAGS
+        - FIM_MODE
+  - directories:
+      value: "/recursiondir_no_tag"
+      attributes:
+        - recursion_level: "2"
+        - FIM_MODE
+# conf 7
```

Comment:
conf 7 is not setting recursion_level. As of right now, this conf is exactly the same as conf 3. In order to check recursion level 0 I would specifically add it as an attribute since we are not ensuring that `syscheck.default_max_depth` is always 0.

From wazuh-documentation:
```
If we don’t want any recursion (just get alerts from the files in the monitored folder), we must set recursion_level to 0.

Warning

If recursion_level is not specified, it will be set to the default value defined by syscheck.default_max_depth in the internal options configuration file.
```